core/unix/module_elf.c

```diff
@@ -1104,7 +1104,6 @@ module_lookup_symbol(ELF_SYM_TYPE *sym, os_privmod_data_t *pd)
 {
     app_pc res;
```

Comment:
Would be nice to add more context in the PR description. Which path does this module_lookup_symbol affect?

Reply:
I've updated the PR description with a bit more context. To save electricity, I haven't yet pushed the empty commit with the similarly updated log message that I have locally.

Reply:
>  I haven't yet pushed the empty commit with the similarly updated log message that I have locally.

There is no need for such a commit as the final merged commit message is taken from the PR description (including edits through the UI): additional commits to this PR branch have zero effect on that final message.